### PR TITLE
Format symbols

### PR DESCRIPTION
**Pull request by BrandonE**
#### Purpose (TL;DR) - mandatory
Properly format [JavaScript symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). Related to [sinon#1974](https://github.com/sinonjs/sinon/issues/1974).

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm test`. This PR adds a symbolic key to one of the objects in a unit test and verifies that it is rendered properly.

#### Checklist for author

- [x] `npm run lint` passes
- [x] References to standard library functions are [cached](https://github.com/sinonjs/sinon/pull/1523).

### CODE REVIEW

File: lib/formatio.js [hunk 1]
```diff
@@ -58,6 +58,10 @@ function ascii(f, object, processed, indent) {
         return processed || quote ? "\"" + object + "\"" : object;
     }
 
+    if (typeof object === "symbol") {
+        return object.toString();
```
Comment (mantoni) [inline lib/formatio.js]: Can you add a separate test for this case please?

Comment (BrandonE) [reply lib/formatio.js]: Done!

**Comment by mantoni** [comment]
Released in `v.3.2.0`.

**Comment by mroderick** [comment]
Very nice!

**Comment by bhousel** [comment]
Ahh it looks like this change is triggering failures on engines that don't support `Object.getOwnPropertySymbols`

I noticed this because our project uses PhantomJS for our tests, but IE11 support would also be an issue.
We can polyfill this as a workaround, but thought you'd want to know!
https://github.com/openstreetmap/iD/issues/6001

**Comment by mantoni** [comment]
Thank you for letting us know @bhousel! We will run into the same issue with the Sinon test suite. I didn't think of IE 11 when reviewing. I'll prepare a patch.

**Pull request merged**